airflow/utils/log/secrets_masker.py

```diff
@@ -34,6 +34,7 @@
 DEFAULT_SENSITIVE_FIELDS = frozenset(
     {
         'access_token',
+        'access_key',
```

Comment:
Generally thinking if "key" should be added? This would be the 4th entry with "key" in the name and generic "secret" and "token" are also in this set. But perhaps it is _too_ general of a term.

Reply:
I'm slightly sceptical if we should even have "access_key" in this list -- at least for AWS connections (the most common source of this) the access_key is not private.

Reply:
Yep, fair point. Although when the AWS Access Key is written to the logs I imagine it's not masked since its specified as `login` in the connection. Only if the access key is added to `Extras` is the value masked (although the [connection doc](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html) for AWS doesn't specify using `Extras` in this manner) -- seems unlikely this approach is used.

Reply:
I wonder if this is a "bug" in WASM connection, and it should be using password for the secret key here?

(A long time ago AWS _used_ to use extra instead of login/password)

Reply:
Two other thoughts though:
- Is "shared_access_key" a better entry in the list?
- Does the `{base.py:79} INFO - Using connection to: id: wasb_default. Host: https:...` line need to be so verbose? Is shortening this to `INFO - Using connection to: id: wasb_default. ` sufficient?

Reply:
I'll close this PR and open an issue to bring in some discussion about what a proper fix could/should be. (Unless you think updates related to any of the above 2 questions would suffice.)

>I wonder if this is a "bug" in WASM connection, and it should be using password for the secret key here?

Obviously some tradeoff between reusing a connection field vs making the connection form more straightforward. Has there typically been deference in a certain direction historically?